src/main/resources/org/broadinstitute/hellbender/utils/helpTemplates/generic.index.template.html

```diff
@@ -1,66 +1,89 @@
-                                </#if>
-                            </tr>
-                        </#if>
-                    </#list>
-                </table>
+        <div class="accordion-group">
+            <div class="accordion-heading">
+                <a class="accordion-toggle" data-toggle="collapse" data-parent="#index" href="#${group.id}">
+                    <h4>${group.name}</h4>
+                </a>
+            </div>
+            <div class="accordion-body collapse" id="${group.id}">
+                <div class="accordion-inner">
+                    <p class="lead">${group.summary}</p>
+                    <table class="table table-striped table-bordered table-condensed">
+                        <tr>
+                            <th>Name</th>
+                            <th>Summary</th>
+                        </tr>
+                        <#list data as datum>
+                            <#if datum.group == group.name>
+                                <tr>
+                                    <td><a href="${datum.filename}">${datum.name}</a></td>
+                                    <#if datum.beta?? && datum.beta == "true">
+                                        <td>**BETA** ${datum.summary}</td>
+                                        <#else>
+                                            <td>${datum.summary}</td>
+                                    </#if>
+                                </tr>
+                            </#if>
+                        </#list>
+                    </table>
+                </div>
             </div>
         </div>
-    </div>
 </#macro>
 
-<h1 id="top">GATK Tool Documentation Index
-    <small>${version}</small>
-</h1>
-<div class="accordion" id="index">
-    <#assign seq = ["engine", "tools", "other", "utilities"]>
-	<#list seq as supercat>
-		<br />
-		<#list groups?sort_by("name") as group>
-			<#if group.supercat == supercat>
-				<@emitGroup group=group/>
-			</#if>
-		</#list>
-	</#list>
-</div>
+        <div class="row-fluid">
+            <div class="span6">
+                <h1 id="top"><i class='<?php print ico::toolDocsIcon; ?>'></i> Tool Documentation Index</h1>
+            </div>
+            <div class="span6">
+                <div class="btn-group pull-right">
+                    <a class="btn btn-warning dropdown-toggle" data-toggle="dropdown" href="#">
+                        Version ${version}
+                        <span class="caret"></span>
+                    </a>
+                    <ul class="dropdown-menu">
+                        <?php foreach($dirs as $dir) {
+                             print "<li><a tabindex='-1' href='../".$dir."'>".$dir."</a></li>";
+                        } ?>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <h1><small>Showing docs for version ${version} | The latest version is <?php print $curr_version; ?></small></h1>
```

Comment:
Ah yes this is entirely due to the php. It doesn't get evaluated in html files; it only does in php files processed through a server-side interpreter and the browser tries to just ignore it, but the sequence of brackets apparently throws off the html interpreter in the browser. I may be able to switch to a different syntax that will just hide all the bits that we can't show correctly without php. 

Good to know about the extension switching, thanks -- it would be ideal to be able to set a gradle switch in the command line, so we can set the release build to emit the html version for packaging and the php version for publishing to the website.